app/controllers/api_controller.rb

```diff
@@ -30,7 +34,7 @@ def team
 
   def messages
     @messages ||=
-      { errors: [], info: [] }
+      { errors: [], infos: [], successes: [] }
```

Comment:
use same error types as we use in flash messages: error, notice, alert, success
btw: info or information is uncountable :)